File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/StaffDashboardStatsResponse.java
package com.cinema.ticket_booking.data.model.response;

import com.google.gson.annotations.SerializedName;

public class StaffDashboardStatsResponse {

    @SerializedName("totalShowtimesToday")
    public long totalShowtimesToday;

    @SerializedName("ticketsCheckedToday")
    public long ticketsCheckedToday;

    @SerializedName("ticketsCheckedThisMonth")
    public long ticketsCheckedThisMonth;
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/StaffDashboardStatsResponse.kt
package com.cinema.ticket_booking.data.model.response

import com.google.gson.annotations.SerializedName

data class StaffDashboardStatsResponse(
    @SerializedName("totalShowtimesToday")
    @JvmField val totalShowtimesToday: Long,

    @SerializedName("ticketsCheckedToday")
    @JvmField val ticketsCheckedToday: Long,

    @SerializedName("ticketsCheckedThisMonth")
    @JvmField val ticketsCheckedThisMonth: Long
)
